gossip/gossip.go

```diff
@@ -90,7 +90,7 @@ const (
 	defaultNodeCount = 1000
 
 	// ttlNodeIDGossip is time-to-live for node ID -> address.
-	ttlNodeIDGossip = 0 * time.Second
+	ttlNodeIDGossip = 0
```

Comment:
(Outside of `const` declarations, though, I agree that it's a little nicer in most cases to use `0` instead of `0 * time.Second`).